packages/cli/tests/cli/add/subgraph/networks.json

```diff
@@ -0,0 +1,7 @@
+{
+  "test": {
```

Comment:
same here

Reply:
This File is needed because the add commands updates the networks.json file